src/cr/cube/stripe.py

```diff
@@ -52,12 +65,25 @@ def rows_including_hidden(self):
 
     @lazyproperty
     def table_base_unpruned(self):
-        """Hmm, weird 1D ndarray with same int value repeated for each row."""
+        """np.int64 scalar or 1D np.int64 ndarray of table base including "empty" items.
+
+        Value is an np.int64 scalar except when this stripe is multiple-response (MR).
+        This value is a 1D ndarray of np.int64 for an MR stripe.
+        """
         return self._base_stripe.table_base
 
     @lazyproperty
     def table_margin_unpruned(self):
-        """Hmm, weird 1D ndarray with same float value repeated for each row."""
+        """np.float/int64 scalar or 1D np.float/int64 ndarray of table margin.
```

Comment:
Maybe add an explanation as to why we need this at all? Normally, you'd want values that are either pruned, or not, according to transformations. So in which case we'd need this? It's not necessary, but might be instructive for a doc-string reader.